lib/models.py

```diff
@@ -355,7 +355,7 @@ def approved_and_ranked(self, proposal_quorum, next_superblock_max_budget):
         return ranked
 
     @classmethod
-    def expired(self, dashd):
+    def expired(self):
```

Comment:
The call to it is here for reference: https://github.com/dashpay/sentinel/pull/16/files#diff-58a68114486f7532a706cd1aac030594R65